From cf8136a8cfcdc04a087b01d586140c457c69f94c Mon Sep 17 00:00:00 2001
From: Pankit Garg <pankit.garg@nxp.com>
Date: Fri, 11 Sep 2020 14:13:50 +0530
Subject: [PATCH 4/5] ls1046awrdb: RN_FFSSPPPN_1133_5577: Modify rcw fields

Make CGA_PLL2_RAT=10 to generate 166Mhz for HS200
Make TVDD_VSEL=0(1.8V)
Make SRDS_REFCLK_SEL_S2=1 as PLL2 to be sourced from SD2_REF_CLK1_P/N
Make SPI_EXT=0 to support SPI for SLIC

Signed-off-by: Pankit Garg <pankit.garg@nxp.com>
---
 ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot.rcw   | 7 ++++---
 .../RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot_sben.rcw      | 7 ++++---
 ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot.rcw   | 8 +++++---
 .../RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot_sben.rcw      | 8 +++++---
 ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_qspiboot.rcw   | 7 ++++---
 ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot.rcw     | 7 ++++---
 .../RN_FFSSPPPN_1133_5577/rcw_1800_sdboot_sben.rcw        | 7 ++++---
 7 files changed, 30 insertions(+), 21 deletions(-)

diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot.rcw
index 7f41337..e8a6c03 100644
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot.rcw
+++ b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot.rcw
@@ -66,7 +66,8 @@
 SYS_PLL_RAT=6
 MEM_PLL_RAT=16
 CGA_PLL1_RAT=16
-CGA_PLL2_RAT=14
+CGA_PLL2_RAT=10
+SRDS_REFCLK_SEL_S2=1
 SRDS_PRTCL_S1=4403
 SRDS_PRTCL_S2=21879
 SRDS_PLL_REF_CLK_SEL_S1=1
@@ -80,7 +81,7 @@ IFC_MODE=37
 EC2=1
 HWA_CGA_M1_CLK_SEL=6
 DRAM_LAT=1
-SPI_EXT=1
+SPI_EXT=0
 UART_BASE=7
 IFC_GRP_A_BASE=1
 IFC_GRP_A_EXT=1
@@ -90,7 +91,7 @@ IFC_GRP_E1_BASE=1
 IFC_GRP_E1_EXT=1
 IFC_GRP_F_EXT=1
 IRQ_OUT=1
-TVDD_VSEL=1
+TVDD_VSEL=0
 DVDD_VSEL=2
 EVDD_VSEL=2
 IIC2_EXT=1
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot_sben.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot_sben.rcw
index 942eca8..e0c5b9d 100644
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot_sben.rcw
+++ b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1600_qspiboot_sben.rcw
@@ -66,7 +66,8 @@
 SYS_PLL_RAT=6
 MEM_PLL_RAT=16
 CGA_PLL1_RAT=16
-CGA_PLL2_RAT=14
+CGA_PLL2_RAT=10
+SRDS_REFCLK_SEL_S2=1
 SRDS_PRTCL_S1=4403
 SRDS_PRTCL_S2=21879
 SRDS_PLL_REF_CLK_SEL_S1=1
@@ -82,7 +83,7 @@ IFC_MODE=37
 EC2=1
 HWA_CGA_M1_CLK_SEL=6
 DRAM_LAT=1
-SPI_EXT=1
+SPI_EXT=0
 UART_BASE=7
 IFC_GRP_A_BASE=1
 IFC_GRP_A_EXT=1
@@ -92,7 +93,7 @@ IFC_GRP_E1_BASE=1
 IFC_GRP_E1_EXT=1
 IFC_GRP_F_EXT=1
 IRQ_OUT=1
-TVDD_VSEL=1
+TVDD_VSEL=0
 DVDD_VSEL=2
 EVDD_VSEL=2
 IIC2_EXT=1
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot.rcw
index 4588300..c8235e2 100644
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot.rcw
+++ b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot.rcw
@@ -66,7 +66,8 @@
 SYS_PLL_RAT=7
 MEM_PLL_RAT=21
 CGA_PLL1_RAT=18
-CGA_PLL2_RAT=14
+CGA_PLL2_RAT=10
+SRDS_REFCLK_SEL_S2=1
 SRDS_PRTCL_S1=4403
 SRDS_PRTCL_S2=21879
 SRDS_PLL_REF_CLK_SEL_S1=1
@@ -80,7 +81,8 @@ IFC_MODE=64
 EC2=1
 HWA_CGA_M1_CLK_SEL=6
 DRAM_LAT=1
-SPI_EXT=1
+SPI_EXT=0
+SPI_BASE=1
 UART_BASE=7
 IFC_GRP_A_BASE=1
 IFC_GRP_A_EXT=1
@@ -90,7 +92,7 @@ IFC_GRP_E1_BASE=1
 IFC_GRP_E1_EXT=1
 IFC_GRP_F_EXT=1
 IRQ_OUT=1
-TVDD_VSEL=1
+TVDD_VSEL=0
 DVDD_VSEL=2
 EVDD_VSEL=0
 IIC2_EXT=0
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot_sben.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot_sben.rcw
index 2e7e0a7..e8000bd 100644
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot_sben.rcw
+++ b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_emmcboot_sben.rcw
@@ -66,7 +66,8 @@
 SYS_PLL_RAT=7
 MEM_PLL_RAT=21
 CGA_PLL1_RAT=18
-CGA_PLL2_RAT=14
+CGA_PLL2_RAT=10
+SRDS_REFCLK_SEL_S2=1
 SRDS_PRTCL_S1=4403
 SRDS_PRTCL_S2=21879
 SRDS_PLL_REF_CLK_SEL_S1=1
@@ -81,7 +82,8 @@ BOOT_HO=1
 IFC_MODE=64
 HWA_CGA_M1_CLK_SEL=6
 DRAM_LAT=1
-SPI_EXT=1
+SPI_EXT=0
+SPI_BASE=1
 UART_BASE=7
 IFC_GRP_A_BASE=1
 IFC_GRP_A_EXT=1
@@ -91,7 +93,7 @@ IFC_GRP_E1_BASE=1
 IFC_GRP_E1_EXT=1
 IFC_GRP_F_EXT=1
 IRQ_OUT=1
-TVDD_VSEL=1
+TVDD_VSEL=0
 DVDD_VSEL=2
 EVDD_VSEL=0
 IIC2_EXT=0
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_qspiboot.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_qspiboot.rcw
index 4aa60fc..2c9fd44 100644
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_qspiboot.rcw
+++ b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_qspiboot.rcw
@@ -66,7 +66,8 @@
 SYS_PLL_RAT=7
 MEM_PLL_RAT=21
 CGA_PLL1_RAT=18
-CGA_PLL2_RAT=16
+CGA_PLL2_RAT=10
+SRDS_REFCLK_SEL_S2=1
 SRDS_PRTCL_S1=4403
 SRDS_PRTCL_S2=21879
 SRDS_PLL_REF_CLK_SEL_S1=1
@@ -80,7 +81,7 @@ IFC_MODE=37
 EC2=1
 HWA_CGA_M1_CLK_SEL=6
 DRAM_LAT=1
-SPI_EXT=1
+SPI_EXT=0
 UART_BASE=7
 IFC_GRP_A_BASE=1
 IFC_GRP_A_EXT=1
@@ -90,7 +91,7 @@ IFC_GRP_E1_BASE=1
 IFC_GRP_E1_EXT=1
 IFC_GRP_F_EXT=1
 IRQ_OUT=1
-TVDD_VSEL=1
+TVDD_VSEL=0
 DVDD_VSEL=2
 EVDD_VSEL=2
 IIC2_EXT=1
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot.rcw
index 263c5fd..1a69ab3 100644
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot.rcw
+++ b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot.rcw
@@ -66,7 +66,8 @@
 SYS_PLL_RAT=7
 MEM_PLL_RAT=21
 CGA_PLL1_RAT=18
-CGA_PLL2_RAT=14
+CGA_PLL2_RAT=10
+SRDS_REFCLK_SEL_S2=1
 SRDS_PRTCL_S1=4403
 SRDS_PRTCL_S2=21879
 SRDS_PLL_REF_CLK_SEL_S1=1
@@ -80,7 +81,7 @@ IFC_MODE=64
 EC2=1
 HWA_CGA_M1_CLK_SEL=6
 DRAM_LAT=1
-SPI_EXT=1
+SPI_EXT=0
 UART_BASE=7
 IFC_GRP_A_BASE=1
 IFC_GRP_A_EXT=1
@@ -90,7 +91,7 @@ IFC_GRP_E1_BASE=1
 IFC_GRP_E1_EXT=1
 IFC_GRP_F_EXT=1
 IRQ_OUT=1
-TVDD_VSEL=1
+TVDD_VSEL=0
 DVDD_VSEL=2
 EVDD_VSEL=2
 IIC2_EXT=1
diff --git a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot_sben.rcw b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot_sben.rcw
index c8eed8c..5d435ec 100644
--- a/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot_sben.rcw
+++ b/ls1046awrdb/RN_FFSSPPPN_1133_5577/rcw_1800_sdboot_sben.rcw
@@ -66,7 +66,8 @@
 SYS_PLL_RAT=7
 MEM_PLL_RAT=21
 CGA_PLL1_RAT=18
-CGA_PLL2_RAT=14
+CGA_PLL2_RAT=10
+SRDS_REFCLK_SEL_S2=1
 SRDS_PRTCL_S1=4403
 SRDS_PRTCL_S2=21879
 SRDS_PLL_REF_CLK_SEL_S1=1
@@ -82,7 +83,7 @@ IFC_MODE=64
 EC2=1
 HWA_CGA_M1_CLK_SEL=6
 DRAM_LAT=1
-SPI_EXT=1
+SPI_EXT=0
 UART_BASE=7
 IFC_GRP_A_BASE=1
 IFC_GRP_A_EXT=1
@@ -92,7 +93,7 @@ IFC_GRP_E1_BASE=1
 IFC_GRP_E1_EXT=1
 IFC_GRP_F_EXT=1
 IRQ_OUT=1
-TVDD_VSEL=1
+TVDD_VSEL=0
 DVDD_VSEL=2
 EVDD_VSEL=2
 IIC2_EXT=1
-- 
2.17.1

